e2e-tests/version-service/run

```diff
@@ -179,7 +179,7 @@ main() {
 	IMAGE_PGBADGER="percona/percona-postgresql-operator:$PGO_TAG-ppg$PG_VER-pgbadger"
 	spinup_pgcluster "${cluster}" "${test_dir}/conf/${PGO_TAG}_cr.yaml"
 
-	apply_version ${cluster} '9.9.9' "disabled" "${PGO_TAG}-${PG_VER}" "${VS_ENDPOINT}" 1
+	apply_version ${cluster} '9.9.9' "disabled" "${PGO_TAG}-${PG_VER}" "${VS_ENDPOINT}" 4
```

Comment:
I see that target revision is incrementing by one on each update attempt, however how it could be possible when we are removing perconapgcluster object? Is the revision stored somewhere even after perconapgcluser removal?

Reply:
It's not a `revision`, it's number of "Smart update finished" messages in operator logs. This solution is working for now, but we need to have special status in our CR for updating